src/modules/endpoint/DataContainer.js

```diff
@@ -101,5 +101,15 @@ export interface DataContainer {
   /**
    * Stores the last usage of the passed city
    */
-  storeLastUsage (city: string, peeking: boolean): Promise<void>
+  storeLastUsage (city: string, peeking: boolean): Promise<void>,
+
+  /**
+   * Deletes all files that are stored offline
+   */
+  clearOfflineStorage (): Promise<void>,
+
+  /**
+   * Clears all caches
+   */
+  clearCaches (): void
```

Comment:
I think this should be named like:

```suggestion
  clearCache (): void
```

because we only have one cache.

Reply:
I just read the implementation: I think this needs better documentation as it clears the in-memory cache and not the filesystem-level cache.

Reply:
Done.